src/PAMI/Message/Action/AgentLogoffAction.php

```diff
@@ -46,8 +47,8 @@ class AgentLogoffAction extends ActionMessage
     /**
      * Constructor.
      *
-     * @param string $agent Agent ID of the agent to log off.
-     * @param string $soft  Set to true to not hangup existing calls.
+     * @param string  $agent Agent ID of the agent to log off.
+     * @param boolean $soft  Set to true to not hangup existing calls.
```

Comment:
Waarom gebruik je hier boolean ipv bool?

Reply:
Gefixed in de nieuwe commit.